CQUI.modinfo

```diff
@@ -1,12 +1,12 @@
 <?xml version="1.0" encoding="utf-8"?>
-<Mod id="1d44b5e7-753e-405b-af24-5ee634ec8a01" version="2.1">
+<Mod id="1d44b5e7-753e-405b-af24-5ee634ec8a01" version="2.2">
```

Comment:
ModBuddy GUI only shows integer version numbers. It allows you to input floats but then rounds it and shows whole number. Building the mod shows whatever version you actually typed in the box. Just FYI, something I noticed. Not sure if we should do anything about it though.

Reply:
I guess just for our use, I suppose?  Yeah if it doesn't do anything I don't know how important it'd be...